<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caseInfoWizardPage</class>
 <widget class="QWizardPage" name="caseInfoWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* CSS Styles used throughout the Avinger code base. Some minor variations may exist. */

QLabel {
	font: 16px &quot;DinPRO-medium&quot;;
    color: rgb(191, 191, 191 );
}

/* Pushbuttons: */
QPushButton {
   background: qlineargradient(spread:pad, x1:1, y1:0.847, x2:1, y2:0, stop:0 rgba(97,  97, 97, 255), stop: 1 rgba(145, 144, 144, 255));
 
  border: 1px black;
  color: white;
  font: 15px &quot;DinPRO-bold&quot;;
  min-width: 164px;
  max-width: 164px;
  min-height: 42px;
  margin: 16px;
  text-align: left;
  padding-left: 15px;
}

QPushButton::hover {
  background: qlineargradient(spread:pad, x1:1, y1:0.847, x2:1, y2:0, stop:0 rgba(144, 144, 145, 255) , stop:1 rgba(198, 197, 197, 255));
  color: black;
}

 QPushButton:default {
 background: qlineargradient(spread:pad, x1:1, y1:0.847, x2:1, y2:0, stop:0  rgba(83, 132, 177, 255) stop:1 rgba(150, 192, 230, 255));
     border: 1px solid black; /* make the default button prominent */
	 font: 15px &quot;DinPro-bold&quot;;
 }

QPushButton:disabled {        /* I don't know what this should look like, probably a darker version of normal */
    border: 1px solid black; 
}

/* Dialog boxes: */

QDialog {   
 font: 18px &quot;DinPRO-regular&quot;;
 color: white;
 background-color: #909090;
 border: 1px solid #909090;
 border-radius: 5px;        /* This doesn't work */
}

/* List views and edits boxes: */

QLineEdit, QListView, QPlainTextEdit {
    color: #808080;
	font: 18px DinPro-medium;
    selection-color: white;   
    border: 2px groove white;
    border-radius: 4px;
}

/* Sliders: */

QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 5px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #8cb7de, stop: 1 #5788b4);
border: 1px solid #5788b4;
height: 10px;
border-radius: 6px;
}

QSlider::add-page:horizontal {
background: #444444;
border: 1px solid #444444;
height: 10px;
border-radius: 6px;
}

QSlider::handle:horizontal {
  image: url(:images/handle_normal.png);
  margin: -15px 0;
}

QSlider::handle:horizontal:hover {
	image: url(:images/handle_rollover.png);
    margin: -15px 0;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

/* Check boxes: */

QCheckBox {
	spacing: 5px;
}

QCheckBox::indicator {
	border: #909090;
    background: #909090;
	width: 20px;
	height: 20px;
	margin-top: 4px;
}

QCheckBox::indicator:unchecked {
	image: url(:images/checkBox_normal.png);
}

QCheckBox::indicator:checked {
	image: url(:images/checkBox_active.png);
}

/* Group boxes: */

QGroupBox {
    font: 20px DINPro-medium;
     color: rgba(173, 171, 171, 255);
     background-color: #909090;
     border: 2px solid #909090;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }
 

QGroupBox::title {
      subcontrol-origin: margin;
      subcontrol-position: top left; /* position at the top center */
      padding: 0 3px;
}

/* Spin Boxes: */

QAbstractSpinBox {
	font: 18px DinPRO-medium;
	border: 0;
    border-radius: 5px;
}

QAbstractSpinBox::up-button {
	border: 0;
   background: rgba(233,233,235,255);
   border-top-right-radius: 8px;
   margin-top: -2px;
   margin-bottom: -1px;
   padding-bottom: -5px;
   width: 35px;
}

QAbstractSpinBox::up-arrow  {
  image: url(:/images/arrow_up_normal.png);
}

QAbstractSpinBox::up-arrow:hover {
  image: url(:images/arrow_up_rollover.png);
}

QAbstractSpinBox::down-button {
   border: 0;
   background: rgba(233,233,235,255);
   border-bottom-right-radius: 8px;
   width: 35px;
   padding-top: -5px;
   margin-bottom: -2px;
   margin-top: -1px;
}

QAbstractSpinBox::down-arrow:hover {
  image: url(:/images/arrow_down_rollover.png);
}

QAbstractSpinBox::down-arrow {
  image: url(:/images/arrow_down_normal.png);
}

/* Combo Boxes */

QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 30px;
      border: 0px;
     border-radius: 4px;
   background: qlineargradient(spread:pad, x1:1, y1:0.847, x2:1, y2:0, stop:0 rgba(97,  97, 97, 255), stop:1 rgba(145, 144, 144, 255));
	margin: -2px;
 }

 QComboBox::down-arrow {
     image: url(:images/arrow_down_normal.png);
}

QComboBox::down-arrow:hover {
	image: url(:images/arrow_down_rollover.png);
}

 QComboBox::down-arrow:on { /* shift the arrow when popup is open */
     top: 1px;
     left: 1px;
 }</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="patientIdLabel">
        <property name="text">
         <string>PATIENT ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="patientIdField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>530</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Patient Name or ID.</string>
        </property>
        <property name="whatsThis">
         <string>The Site ID is combined with the Subject ID to create the Case ID.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="doctorLabel">
        <property name="text">
         <string>DOCTOR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="doctorField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>530</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>LOCATION</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="locationField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>530</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="laserLabel">
        <property name="font">
         <font>
          <family>DinPRO-medium</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 32px;</string>
        </property>
        <property name="text">
         <string>Laser control is disabled!</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="notesLabel">
        <property name="text">
         <string>NOTES</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPlainTextEdit" name="notesField">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>530</width>
          <height>140</height>
         </size>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>VERSION</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="versionField">
        <property name="text">
         <string notr="true">0.0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="instructionsText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>196</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 25px &quot;DinPRO-medium&quot;;</string>
   </property>
   <property name="text">
    <string>Please enter the case information. Patient ID is required.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>patientIdField</tabstop>
  <tabstop>doctorField</tabstop>
  <tabstop>locationField</tabstop>
  <tabstop>notesField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
